utils/download_hplt.py

```diff
@@ -0,0 +1,85 @@
+import argparse
```

Comment:
Suggestion: can you add a docstring explaining the state of this script, e.g. something like what you wrote in the PR.